scripts/update_chrome_shortcut.py

```
@@ -68,32 +104,44 @@ def update_chrome_shortcuts_info() -> tuple[list[Path], str]:
     """Return True if any Chrome shortcuts are missing the debug flag."""
-    if os.name != "nt" or win32com is None:
+    if os.name == "nt" and win32com is not None:
+        shell = win32com.client.Dispatch("WScript.Shell")
+        locations = [
+            Path(os.environ.get("USERPROFILE", "")) / "Desktop",
+            Path(os.environ.get("APPDATA", ""))
+            / "Microsoft"
+            / "Windows"
+            / "Start Menu"
+            / "Programs",
+            Path(os.environ.get("ProgramData", ""))
```

Ruff (0.11.9)

Use capitalized environment variable `PROGRAMDATA` instead of `ProgramData`

(SIM112)